cvgui.py

```diff
@@ -710,20 +708,34 @@ def printKeyBindings(self, key=None):
             fks = ', '.join([kcd for kcd in funs[fn]])             # keep track of functions with multiple keybindings so we know how to format the output
             keyCodeLen = len(fks) if len(fks) > keyCodeLen else keyCodeLen
         
+        #set maximum line for description
+        #get terminal_size
+        rows, columns = os.popen('stty size', 'r').read().split()
+        docLen = int(columns) - keyCodeLen - funLen - 8
+        
         # create string templates for printing
-        tStr = '{:' + str(funLen) + '} | {:' + str(keyCodeLen) + '} | {}'                # template string (for formatting output into columns)
+        tStr = '{:' + str(funLen) + '} | {:' + str(keyCodeLen) + '} | {:' + str(docLen) + '} |'           # template string (for formatting output into columns)
         
         # print header string with table formatting
         print tStr.format(funStr, keyStr, docStr)
         print tStr.format(''.join(['-' for i in range(0,funLen)]),''.join(['-' for i in range(0,keyCodeLen)]),''.join(['-' for i in range(0,docLen)]))
         
-        # go through all the known keybindins and print their info
+        # go through all the known keybindings and print their info
         for fn in sorted(funs.keys()):
             ks = ', '.join([kcd for kcd in funs[fn]])
             doc = getattr(self, fn).__doc__
             ds = ' '.join([s for s in map(lambda s: s.strip(), doc.splitlines()) if len(s) > 0]) if doc is not None else ''
-            print tStr.format(fn, ks, ds)
-        
+            
+            # check if the length of description exceed the terminal_width
+            if len(ds) > docLen:
+                lines=1
```

Comment:
Usually I like to add spaces around the equal sign for assignment/incrementing (but not for keyword arguments, as you see in various places).